lib/puppet/provider/sensor_download/sensor_download.rb

```diff
@@ -17,7 +17,12 @@ def destroy
   end
 
   def exists?
-    falcon_version = Facter.value('falcon').fetch('version', :purged)
+    falcon_version = if Facter.value('falcon').nil?
```

Comment:
`Facter.value('falcon')&.fetch('version') || :absent` may work as well and potentially disgust some python people.

Reply:
Updated to use your magic :)